docs/how-to/uc-image-creation.rst

```diff
@@ -161,14 +204,23 @@ The `gadget snap`_ documentation is a recommended read before starting.
 This is best done by forking an existing reference gadget, then changing it for our purpose.
 For example, there is the `pc gadget`_ which is suitable for most AMD64 platforms, and the `pi gadget`_ which is meant for Raspberry Pis.
 
-This section uses the core22 pc gadget snap as an example for creating a custom gadget snap.
+Inside the project directory, clone the specific branch of the pc-gadget and enter the repository:
```

Comment:
I think it's missing a word?
"...clone the specific branch of the pc-gadget repository and..."